frontend/views/containers/user-settings/Acknowledgements.vue

```diff
@@ -0,0 +1,145 @@
+<template lang='pug'>
+  .settings-container
+    section.card
+      .c-header
+        p.c-instructions
+          i18n(
+            :args='{ a_: okTurtlesPageTag, _a: "</a>" }'
+          ) Group Income is developed by the {a_}okTurtles Foundation{_a} and is funded entirely by individual contributions.
+      i18n.link(tag='button' @click='openSupport') Support this project
+
+      .c-wrapper(
+        v-for='(item, index) in list'
+        :key='index'
+      )
+        .c-title(tag='h2') {{ item.legend }}
+        .c-smaller-title(
+          v-for='(link, idx) in item.links'
+          :key='idx'
+        ) {{ link.title }}
+          a.sublink(@click='redirectTo(link.url)') {{ link.name }}
+</template>
+
+<script>
+import safeLinkTag from '@view-utils/safeLinkTag.js'
+import { L } from '@common/common.js'
+
+export default ({
+  name: 'Acknowledgements',
+  data () {
+    return {
+      okTurtlesPageTag: safeLinkTag('OKTURTLES_PAGE'),
+      list: [{
+        legend: L('Libraries'),
+        links: [
+          {
+            title: 'Vue',
+            name: 'vuejs.org',
+            url: 'https://vuejs.org'
+          },
+          {
+            title: 'Grunt',
+            name: 'gruntjs.com',
+            url: 'https://gruntjs.com'
+          },
+          {
+            title: 'Cypress',
+            name: 'cypress.io',
+            url: 'https://www.cypress.io'
+          },
+          {
+            title: 'Hapi',
+            name: 'hapi.dev',
+            url: 'https://hapi.dev'
+          }
+        ]
+      }, {
+        legend: L('Illustrations'),
+        links: [
+          {
+            title: 'Voting',
+            name: 'By Nikita Kozin',
+            url: '#'
+          },
+          {
+            title: 'Broken Link',
+            name: 'By HeadsOfBirds',
+            url: '#'
+          },
+          {
+            title: 'Send',
+            name: 'By Gregor Cresnar',
+            url: '#'
+          },
+          {
+            title: 'Hello',
+            name: 'By Allie',
+            url: '#'
+          },
+          {
+            title: 'Conversation',
+            name: 'By Gregor Cresnar',
+            url: '#'
+          },
+          {
+            title: 'Vote',
+            name: 'By HeadsOfBirds',
+            url: '#'
```

Comment:
If you aren't able to find links to these, could you comment them out for now?

Reply:
Okay.